src/libs/pool-data-provider/hooks/use-stake-data-with-rpc.tsx

```diff
@@ -37,10 +37,8 @@ export function useStakeDataWithRpc(
   const [stakeData, setStakeData] = useState<StakesData>();
   const [usdPriceEth, setUsdPriceEth] = useState<string>('0');
 
-  const loadStakeData = async (
-    userAddress: string = ethers.constants.AddressZero,
-    helperAddress: string
-  ) => {
+  const loadStakeData = async (_userAddress: string | undefined, helperAddress: string) => {
```

Comment:
useraddress is not undefined, but an empty string if not set so this default value doesn't work